service-coordtransform/src/main/java/fi/nls/paikkatietoikkuna/coordtransform/CoordinateTransformationActionHandler.java

```diff
@@ -351,10 +349,13 @@ protected void transform(String sourceCrs, String targetCrs,
                 }
             }
         } catch (UnsupportedEncodingException e){
+            log.warn(e, e.getMessage());
```

Comment:
It's usually a bad idea to log AND throw as it leads to excessive logging. Could the original exception be passed as the cause for ActionParamsException?